From b210b9ba867e92632b9dec358e50dc166952a6ac Mon Sep 17 00:00:00 2001
From: Gustavo Duarte <gduarte@redhat.com>
Date: Mon, 1 Sep 2014 20:15:30 -0400
Subject: [powerpc] perf/hv_gpci: Probe failures use pr_debug(), and padding reduced

Message-id: <1409602534-23951-12-git-send-email-gduarte@redhat.com>
Patchwork-id: 90126
O-Subject: [RHEL7.1 PATCH BZ 947159 11/15] powerpc/perf/hv_gpci: Probe failures use pr_debug(), and padding reduced
Bugzilla: 947159
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=947159
Upstream Status: 0a8cf9e28c094cd934deaedfd3ec07830f2a63cf

commit 0a8cf9e28c094cd934deaedfd3ec07830f2a63cf
Author: Cody P Schafer <cody@linux.vnet.ibm.com>
Date:   Tue Apr 15 10:10:51 2014 -0700

    powerpc/perf/hv_gpci: Probe failures use pr_debug(), and padding reduced

    fixup for "powerpc/perf: Add support for the hv gpci (get performance
    counter info) interface".

    Makes the "not enabled" message less awful (and hidden unless
    debugging).

    Signed-off-by: Cody P Schafer <cody@linux.vnet.ibm.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/perf/hv-gpci.c b/arch/powerpc/perf/hv-gpci.c
index 278ba7b..8fee1dc 100644
--- a/arch/powerpc/perf/hv-gpci.c
+++ b/arch/powerpc/perf/hv-gpci.c
@@ -273,13 +273,13 @@ static int hv_gpci_init(void)
  struct hv_perf_caps caps;
 
  if (!firmware_has_feature(FW_FEATURE_LPAR)) {
-  pr_info("not a virtualized system, not enabling\n");
+  pr_debug("not a virtualized system, not enabling\n");
   return -ENODEV;
  }
 
  hret = hv_perf_caps_get(&caps);
  if (hret) {
-  pr_info("could not obtain capabilities, error 0x%80lx, not enabling\n",
+  pr_debug("could not obtain capabilities, not enabling, rc=%ld\n",
     hret);
   return -ENODEV;
  }
-- 
1.7.1